master/getting-started/kubernetes/installation/index.md

```diff
@@ -12,14 +12,17 @@ information.
 {{site.prodname}} can run on any Kubernetes cluster which meets the following criteria.
 
 - The kubelet must be configured to use CNI network plugins (e.g `--network-plugin=cni`).
-- The kube-proxy must be started in `iptables` proxy mode.  This is the default as of Kubernetes v1.2.0.
+- The kube-proxy must be started in either `iptables` or (beta) `ipvs` proxy
+  mode.  `iptables` is the default as of Kubernetes v1.2.0.  Calico's `ipvs`
+  mode support is currently in beta and it has [some limitations](#ipvs-mode).
+- For kube-proxy `ipvs` mode, Kubernetes v1.9.3 or above is required due to
+  critical bugs in earlier releases.
```

Comment:
Great information! I might suggest a slightly different treatment to make it a little easier to read and understand.
```
kube-proxy must be started in one of the following modes:

   - `iptables` the default as of Kubernetes v1.2.0
   - `ipvs` {{site.prodname}} supports this at the beta level with [some limitations](#ipvs-mode). Requires Kubernetes v1.9.3 or later due to critical bugs in earlier releases.
```